tests/integration/component.go

```diff
@@ -583,13 +583,10 @@ func componentTests(args ...string) {
 			ts := time.Now().UnixNano()
 			contextNumeric, err = ioutil.TempDir("", fmt.Sprint(ts))
 			Expect(err).ToNot(HaveOccurred())
-			os.Mkdir(context, 0750)
 			project = helper.CreateRandProject()
-			helper.Chdir(contextNumeric)
 		})
 		JustAfterEach(func() {
 			helper.DeleteProject(project)
-			helper.Chdir(originalDir)
```

Comment:
@kadel Issue is coming from here
```
Unexpected error:
      
        <*os.PathError | 0xc0000ad650>: {Op: "chdir", Path: "", Err: 0x2}
        chdir : no such file or directory
    occurred
     /go/src/github.com/openshift/odo/tests/helper/helper_filesystem.go:42
```

Reply:
This is the function where the issue hit - https://github.com/openshift/odo/blob/master/tests/helper/helper_filesystem.go#L39